tests/dummy/app/templates/components/file-tree.hbs

```diff
@@ -0,0 +1,10 @@
+<div style="padding-left:1em">
```

Comment:
Also please keep css out of handlebars templates, instead use a style.css in this folder https://github.com/CenterForOpenScience/ember-osf/blob/develop/.github/CONTRIBUTING.md#styling-components